static/app/utils/profiling/flamegraph/flamegraphTheme.tsx

```diff
@@ -109,15 +109,15 @@ export const LCH_DARK = {
 };
 
 const SPAN_LCH_LIGHT = {
-  C_0: 0.25,
-  C_d: 0.2,
-  L_0: 0.7,
-  L_d: 0.25,
+  C_0: 0.3,
```

Comment:
Slightly tweaked the colors and the size of the span bar as it was a bit too jammed in there